workflows/carbon_emissions.md

```diff
@@ -238,39 +241,81 @@ When adding emissions inputs in the project properties, URBANopt will apply thes
-  "future_hourly_emissions_mt":  
-    Future emissions in metric tons (mt) calculated based on hourly emissions factor values for the selected future year
-  "historical_annual_emissions_mt": 
-    Historical emissions in metric tons (mt) calculated based on an annual emissions factor value for the selected historical year
-  "historical_hourly_emissions_mt":  
-    Historical emissions in metric tons (mt) calculated based on hourly emissions factor values for the selected historical year
-  "future_annual_emissions_intensity_kg_per_ft2": 
-    Future emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected future year
-  "future_hourly_emissions_intensity_kg_per_ft2": 
-    Future emissions intensity in kg/ft2 calculated based on hourly emissions factor values for the selected future year
-  "historical_annual_emissions_intensity_kg_per_ft2": 
-    Historical emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected historical year
-  "historical_hourly_emissions_intensity_kg_per_ft2":
-    Historical emissions intensity in kg.ft2 calculated based on hourly emissions factor values for the selected historical year
+	"future_annual_electricity_emissions_mt":
+	Future electricity emissions in metric tons (mt) calculated based on an annual emissions factor value for the selected future year
+
+	"future_hourly_electricity_emissions_mt":  
+	Future electricity emissions in metric tons (mt) calculated based on hourly emissions factor values for the selected future year
+
+	"historical_annual_electricity_emissions_mt": 
+	Historical electricity emissions in metric tons (mt) calculated based on an annual emissions factor value for the selected historical year
+
+	"historical_hourly_electricity_emissions_mt":  
+	Historical electricity emissions in metric tons (mt) calculated based on hourly emissions factor values for the selected historical year
+
+	"future_annual_electricity_emissions_intensity_kg_per_ft2": 
+	Future electricity emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected future year
+
+	"future_hourly_electricity_emissions_intensity_kg_per_ft2": 
+	Future electricity emissions intensity in kg/ft2 calculated based on hourly emissions factor values for the selected future year
+
+	"historical_annual_electricity_emissions_intensity_kg_per_ft2": 
+	Historical electricity emissions intensity in kg/ft2 calculated based on an annual emissions factor value for the selected historical year
+
+	"historical_hourly_electricity_emissions_intensity_kg_per_ft2":
+	Historical electricity emissions intensity in kg.ft2 calculated based on hourly emissions factor values for the selected historical year
+
+	"natural_gas_emissions_mt": 
+	Natural Gas emissions in mt calculated by multipling the facility natural gas energy use (MWH) by the natural gas emission factor (277.358126 KG/MWH) and convertd to metric ton(mt)
```

Comment:
```suggestion
	Natural Gas emissions in metric tonnes (mt) were calculated by multiplying the facility's natural gas energy use (MWh) by the natural gas emission factor (277.4 kg/MWH) and converted to metric tonnes (mt)
```
I think tonnes is the right spelling here? Or maybe not. Another opinion is probably needed.